### PR TITLE
Support mod_proxy_wstunnel for Apache 2.4+

### PR DESCRIPTION
**Pull request by tpdownes**
Includes basic documentation. Derived from previous pull requests #1290 and #1325.

### CODE REVIEW

**Comment by tpdownes** [comment]
The first version of this pull request had a couple minor problems. Please make sure you pull from 5eec44e

File: README.md [hunk 1]
```diff
@@ -1636,6 +1637,9 @@ Default values depend on your operating system.
 
 **Note**: There is no official package available for mod\_proxy\_html and thus it must be made available by means outside of the control of the apache module.
 
+##### Class: `apache::mod::proxy_wstunnel`
+Includes the mod\_proxy_\wstunnel module on Apache versions >= 2.4.5. For versions below, no action is taken.
```
Comment (hunner) [inline README.md]: Technically `>= 2.4` not 2.4.5

**Comment by tphoney** [comment]
@tpdownes Thanks for the submission, it would be great if you added a simple test case.

File: README.md (hunk 1 above)
Comment (tpdownes) [inline README.md]: The module doesn't exist below 2.4.5 (unless backported) but `::apache::version` doesn't offer finer resolution. I fixed the README to reflect the actual behavior. A relatively quick scan doesn't reveal any OS-packaged versions that are 2.4.4 or below.

**Comment by tpdownes** [comment]
It would probably take me a day or 2 to figure out how to mimic something like `spec/classes/mod/proxy_html_spec.rb`. It's not that I don't support the idea of unit tests, I'm just not a ruby coder and don't have much experience with the CI system.

**Comment by tphoney** [comment]
@tpdownes do you need any help with the test case ?

**Comment by tpdownes** [comment]
There's no way I'll be getting to this in the next month or two - total reorganization of the data center going on. Baby on way, etc. So, I guess the answer is yes.

**Comment by tphoney** [comment]
@tpdownes good luck on the baby, and the data center. i will get a look at this next week.

**Comment by tphoney** [comment]
@jonnytpuppet this should just need a simple test case, copying what is done for other apache plugins. It  looks like a quick win.

**Comment by DavidS** [comment]
An equivalent class with tests was merged as #1465

**Pull request closed without merging**